proxy/http/HttpSM.cc

```diff
@@ -4175,28 +4175,28 @@ HttpSM::check_sni_host()
             SMDebug("ssl_sni", "[HttpSM::check_sni_host] No SNI for TLS request with hostname %.*s action=%s", host_len, host_name,
                     action_value);
             if (host_sni_policy == 2) {
-              Log::error("%s", lbw()
-                                 .clip(1)
-                                 .print("No SNI for TLS request: connecting to {} for host='{}', returning a 403",
-                                        t_state.client_info.dst_addr, std::string_view{host_name, static_cast<size_t>(host_len)})
-                                 .extend(1)
-                                 .write('\0')
-                                 .data());
+              Error("%s", lbw()
+                            .clip(1)
+                            .print("No SNI for TLS request: connecting to {} for host='{}', returning a 403",
+                                   t_state.client_info.dst_addr, std::string_view{host_name, static_cast<size_t>(host_len)})
+                            .extend(1)
+                            .write('\0')
+                            .data());
```

Comment:
Maybe keep these connection-oriented messages in error.log? That's what I expect in error.log anyway: connection issues.